<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1685</width>
    <height>1153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1621</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Browse</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1571</width>
        <height>991</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>All Resources</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>1360</x>
          <y>850</y>
          <width>111</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>1491</width>
          <height>791</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Books</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>1360</x>
          <y>850</y>
          <width>111</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>1491</width>
          <height>791</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Essays</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Articles</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Periodicals</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Issues</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>1551</width>
        <height>981</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Resource</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>1101</width>
          <height>851</height>
         </rect>
        </property>
        <property name="title">
         <string>Add General Resource</string>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>390</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>80</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>220</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Date of Publish</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>220</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>450</y>
           <width>781</width>
           <height>151</height>
          </rect>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>680</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Add Resource</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>130</y>
           <width>190</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>280</y>
           <width>190</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>280</y>
           <width>190</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Book</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Essay</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Article</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>Periodical</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>Issue</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_15">
     <attribute name="title">
      <string>Search</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1685</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLibrary_resourcses">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionBooks"/>
   </widget>
   <widget class="QMenu" name="menuBooks">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuLibrary_resourcses"/>
   <addaction name="menuBooks"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBooks">
   <property name="text">
    <string>Add Resource</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
